test/utf8/rlexe.xml

```diff
@@ -15,6 +15,13 @@
   </test>
   <test>
     <default>
+      <files>OS_2977448.js</files>
+      <compile-flags>-Intl-</compile-flags>
```

Comment:
Is turning off Intl required? I don't know if it would make a difference in this test since the test is not using Intl or any of the built-ins which plumb into platform APIs.